bpf/attach.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bpf
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+const (
+	BPF_PROG_BINARY_DIR    = "/usr/lib/calico/bpf"
```

Comment:
I think it is good to keep these public, in case some other packages would want to work with these directories. For example both TC and XDP have these directories hard coded. What do you think?

Reply:
noticed one of these is already defined in bpf-def.go will move the other one also to the same file.